### PR TITLE
feat(deno): update spec to 1.17

### PR DESCRIPTION
**Pull request by SeparateRecords**
Only minor updates this time!

### CODE REVIEW

File: src/deno.ts [hunk 1]
```diff
@@ -96,6 +96,18 @@ type ExclusiveOn = {
   exclusiveOn?: string[];
 };
 
+const unsafelyIgnoreCertificateErrorsOption = {
```
Comment (fedeci) [inline src/deno.ts:99]:
```suggestion
const unsafelyIgnoreCertificateErrorsOption: Fig.Option = {
```

**Review by fedeci — APPROVED** [review]

**Comment by fedeci** [comment]
Just one nit, then we can get this merged.

**Comment by SeparateRecords** [comment]
Thank you for that! 😆

**Review by mschrage — APPROVED** [review]
👌

**Pull request merged**